src/utils.js

```diff
@@ -208,7 +208,7 @@ export function transitionOut(el, hide, component, forceSkip = false) {
             ? modifiers.filter((i, index) => index > modifiers.indexOf('out')) : modifiers
 
         transitionHelperOut(el, modifiers, settingBothSidesOfTransition, hide)
-    } else if (attrs.filter(attr => ['leave', 'leave-start', 'leave-end'].includes(attr.value)).length > 0) {
+    } else if (attrs.some(attr => ['leave', 'leave-start', 'leave-end'].includes(attr.value))) {
```

Comment:
In this cases it's faster to use `some` instead of `filter`

You can find the benchmark here:
https://www.measurethat.net/Benchmarks/Show/8658/alpinejs-speed-up-transitionin-transitionout

---

MDN about [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)
> The some() method tests whether at least one element in the array passes the test implemented by the provided function. It returns a Boolean value.

---

MDN about [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)
> The filter() method creates a new array with all elements that pass the test implemented by the provided function.